peachjam/models/attachments.py

```diff
@@ -110,6 +110,9 @@ class SourceFile(AttachmentAbstractModel):
         null=True,
         blank=True,
     )
+    sha256 = models.CharField(
+        "SHA 256", max_length=64, null=True, blank=True, db_index=True
```

Comment:
could we set this as unique and let db validation handle it?

Reply:
That's a good suggestion. Maybe we can follow up once this is deployed.